src/style.css

```diff
@@ -0,0 +1,62 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  width: 140px;
+  height: 160px;
+  background-color: #fff;
+}
+
+.stars {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-basis: 96px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-image: url(./images/star.svg);
+  background-size: auto;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-of-type(1) {
+  background-image: url(./images/star-active.svg);
+  background-size: auto;
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars__star:nth-of-type(-n+2) {
+  background-image: url(./images/star-active.svg);
+  background-size: auto;
+  background-repeat: no-repeat;
+}
+
+.stars--3 .stars__star:nth-of-type(-n+3) {
+  background-image: url(./images/star-active.svg);
+  background-size: auto;
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:nth-of-type(-n+4) {
+  background-image: url(./images/star-active.svg);
+  background-size: auto;
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars__star:nth-of-type(n) {
+  background-image: url(./images/star-active.svg);
+  background-size: auto;
+  background-repeat: no-repeat;
+}
```

Comment:
Don't duplicate these rules. You can combine these selectors (with the comma) and use rules just once.